test/spy-test.js

```diff
@@ -240,7 +240,26 @@
             }
         },
 
-        call: {
+        // test for regression introduced in #692. See #1046 for details
+        "localStorage support": {
+            requiresSupportFor: {
+                browser: typeof window !== "undefined",
+                localStorage: typeof window.localStorage !== "undefined"
+            },
+
+            "can spy on localStorage": function () {
+                var spy = sinon.spy(localStorage, "getItem");
+
+                localStorage.clear();
+                localStorage.setItem("foo", "fooval");
+                localStorage.getItem("foo");
+
+                assert(typeof localStorage.getItem.restore, "function" );
+                assert(spy.firstCall.returnValue === "fooval");
+            }
+        },
+
+        "call": {
```

Comment:
Consistency. All other test names have them, so while it is technically fine, I don't see why it should be treated specially.